pkg/tags/manager.go

```diff
@@ -42,16 +42,14 @@ func init() {
 
 func CloseTagSets() (err error) {
 	mtx.Lock()
-	if tagSets != nil {
```

Comment:
Removing this nil check is the only thing I am uncertain about. I am guessing that `staticcheck` said that this couldn't be nil or something?

Reply:
the range immediately after this check is itself a nil check

staticcheck yelled about it.